package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "@next/font": "13.1.2",
+    "axios": "^1.3.2",
```

Comment:
i don't reject the usage of `axios`, but usually `axios` is used instead of using the native `fetch` api directly. currently you're using both.

Reply:
I am aware of this and don't think its best practice either. I will fix it after the capstone deadline.